templates/frontend/pages/catalogSeriesIndex.tpl

```diff
@@ -0,0 +1,66 @@
+{**
+ * templates/frontend/pages/catalogSeries.tpl
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University Library
+ * Copyright (c) 2003-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @brief Display the page to view books in a series in the catalog.
+ *
+ * @uses $series Series Current series being viewed
+ * @uses $publishedMonographs array List of published monographs in this series
+ * @uses $featuredMonographIds array List of featured monograph IDs in this series
+ * @uses $newReleasesMonographs array List of new monographs in this series
```

Comment:
The `@brief` and `@uses` declarations need to be updated to reflect this template.

Reply:
that is:
 * @brief Display a page with an overview the Book series in the press.
 * @uses $browseSeriesFactory array List of series